### PR TITLE
role & access keys widgets

### PR DESCRIPTION
**Pull request by nirgur**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -7,17 +7,31 @@ import DescopeWc from '@descope/web-component';
...
 	}
 }
 
+type WidgetProps = {
+	logger?: ILogger;
+	tenant: string;
+	widgetId: string;
+	// If theme is not provided - the OS theme will be used
+	theme?: ThemeOptions;
+	// If locale is not provided - the browser's locale will be used
```
Comment (dorsha) [inline src/types.ts:31]: wrong comment

**Review by dorsha — COMMENTED** [review]
Just small commet

File: src/types.ts (hunk 1 above)
Comment (nirgur) [reply src/types.ts]: removed

File: package.json [hunk 2]
```diff
@@ -48,7 +48,9 @@
 		]
 	},
 	"dependencies": {
-		"@descope/user-management-widget": "0.2.3",
+		"@descope/user-management-widget": "0.4.5",
```
Comment (dorsha) [inline package.json:51]: Please use latest

**Review by dorsha — APPROVED** [review]

**Pull request merged**